kube-client/src/discovery/mod.rs

```diff
@@ -138,6 +137,16 @@ impl Discovery {
         self.groups.values()
     }
 
+    /// Returns a sorted vector of all served groups
+    ///
+    /// This vector is in kubectl's normal alphabetical group order
+    pub fn groups_alphabetical(&self) -> Vec<&ApiGroup> {
+        let mut values: Vec<_> = self.groups().collect();
+        // collect to maintain kubectl order of groups
+        values.sort_by_cached_key(|g| &g.name);
+        values
+    }
```

Comment:
King of want to have this extra method since it's the most useful container and most useful ordering, particularly given that doing comparisons via the iterator interface is hard: if you want a sorted vec you are going to need to do a `mut` collect + sort by key or `ApiGroup::alpha_cmp` otherwise.

Reply:
Sure.